shell/pages/about.vue

```diff
@@ -195,6 +196,7 @@ export default {
           <td>
             <a
               v-if="d.imageList"
+              :data-testid="`image_list__${d.label}`"
```

Comment:
This and below should also be more unique. alternatively from tests the above data-testid could be found first and then some other selector is used to find these two

Reply:
@richard-cox I was able to just do `.contains(link text)` to find CLI links since the text is different. However, it is difficult to target the image list 'Download' links because the link text is the same so for those two I've updated the data-testid to be more unique (so for Linux it would be `image_list_download_link__about.os.linux`). let me know if you want me to update further.